src/kernel/arch/x86/paging.zig

```diff
@@ -184,7 +192,7 @@ test "mapDir" {
     var direct = std.heap.DirectAllocator.init();
```

Comment:
Could you add more unit tests for mapDir to test all the return errors/code paths. You can use expectError.